src/api/constants.js

```diff
@@ -0,0 +1,11 @@
+/* @flow strict-local */
+
+/**
+ * The topic servers understand to mean "there is no topic". This should
+ * match
+ *   https://github.com/zulip/zulip/blob/5.0-dev/zerver/lib/actions.py#L2714
+ * or similar logic at the latest `main`.
+ */
+// This is hardcoded in the server, and therefore untranslated; that's
+// zulip/zulip#3639.
+export const NO_TOPIC_TOPIC: '(no topic)' = '(no topic)';
```

Comment:
I think this is one where the type is best written as `string` rather than the specific string: as far as we want the type-checker to be concerned, this is just some string, much like how a timeout duration would be just some number. It's not a string that we want to have participate in determining our types, like the tag strings that we put in the `type` property of a `Message` or an `Action`.